concent_api/core/models.py

```diff
@@ -1,26 +1,29 @@
-from django.db.models import BinaryField
-from django.db.models import BooleanField
-from django.db.models import CharField
-from django.db.models import DateTimeField
-from django.db.models import ForeignKey
-from django.db.models import PositiveSmallIntegerField
-from django.db.models import Manager
-from django.db.models import Model
-from django.db.models import OneToOneField
-from django.db.models import Q
-from django.db.models import QuerySet
-
-from utils.fields     import Base64Field
-from utils.helpers    import is_base64
-
-from .constants       import MESSAGE_TASK_ID_MAX_LENGTH
-
-
-class StoredMessage(Model):
-    type        = PositiveSmallIntegerField()
-    timestamp   = DateTimeField()
-    data        = BinaryField()
-    task_id     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, null = True, blank = True)
+import base64
+
+from django.core.validators import ValidationError
+from django.db              import models
+from django.db.models       import BooleanField
+from django.db.models       import DateTimeField
+from django.db.models       import ForeignKey
+from django.db.models       import Manager
+from django.db.models       import Model
+from django.db.models       import OneToOneField
+from django.db.models       import Q
+from django.db.models       import QuerySet
+
+from utils.fields           import Base64Field
+from utils.fields           import ChoiceEnum
+from utils.helpers          import is_base64
+
+from .constants             import MESSAGE_TASK_ID_MAX_LENGTH
+
+
+class StoredMessage(models.Model):
+    type        = models.PositiveSmallIntegerField()
+    timestamp   = models.DateTimeField()
+    data        = models.BinaryField()
```

Comment:
Strange, it was originally done this way, because my point was to get rid of those imports, so it must get lost somehow during rebases. I will fix it :+1:

Reply:
I want to get rid of all those imports in step 3 and import only models.

Reply:
I think that it's better to import them individually. It was done on purpose like this from the beginning. It makes the model definition more concise and readable because we don't have to insert the `model.` prefix everywhere.

Reply:
:+1: Ok, I didn't know it was on purpose, I will add them back.